cpp/CMakeLists.txt

```diff
@@ -183,22 +198,19 @@ endif (RMM_INCLUDE AND RMM_LIBRARY)
 # - CUDF -------------------------------------------------------------------------------------------
 
 find_path(CUDF_INCLUDE "cudf"
-          HINTS "$ENV{CUDF_ROOT}/cpp/include")
+          HINTS "$ENV{CUDF_ROOT}/include")
 
 find_library(CUDF_LIBRARY "cudf"
              HINTS "$ENV{CUDF_ROOT}/lib")
 
-SET(CUDF_SRC_INCLUDE "$ENV{CUDF_HOME}/cpp/src")
-SET(CUB_INCLUDE "$ENV{CUDF_HOME}/cpp/thirdparty/cub")
-SET(CUDACXX_INCLUDE "$ENV{CUDF_HOME}/cpp/thirdparty/libcudacxx/include")
-SET(CUDF_TEST_INCLUDE "$ENV{CUDF_HOME}/cpp/")
+set(CUDF_TEST_INCLUDE "$ENV{CUDF_HOME}/cpp")
```

Comment:
Reaching into the libcudf source like this makes me cautious. Are these headers not shipped in a libcudf install?

Reply:
They're not, unfortunately. AFAIK cuSpatial is the only external consumer of `libcudftestutil.a` so far.